src/components/offer-card-square/OfferCardSquare.styles.ts

```diff
@@ -0,0 +1,96 @@
+const ellipsisTextStyle = (linesCount: number) => ({
+  display: '-webkit-box',
+  WebkitLineClamp: linesCount,
+  lineClamp: linesCount,
+  WebkitBoxOrient: 'vertical',
+  boxOrient: 'vertical',
+  overflow: 'hidden'
+})
+
+export const styles = {
+  containerCard: {
+    maxWidth: '320px'
+  },
+  container: {
+    position: 'relative'
+  },
+  mainInfo: {
+    root: {
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center'
+    },
+    img: {
+      width: '80px',
+      borderRadius: '50%'
+    },
+    description: {
+      ...ellipsisTextStyle(2),
+      textAlign: 'left'
+    },
+    title: {
```

Comment:
i can`t see where you use this img, description and title styles

Reply:
fixed